website/docs/guides/dependencies.md

```diff
@@ -36,6 +36,45 @@ For example:
 org.postgresql:postgresql:42.2.8
 ```
 
+### Excluding Transitive Dependencies
+
+To exclude a transitive dependency of dependency in a Scala CLI project, use the `exclude` parameter when specifying
+the dependency:
+
+- `exclude=org%%name` - for Scala modules
```

Comment:
maybe just note on what the double `%` stands for here smh... especially since it'd be triple `%` for some cases. So maybe just mention that double `%` is the way for 95% of Scala dependency cases...

Reply:
It seem, that coursier doesn't triple `%` for some cases, it throws the following error when I trying to pass triple `%`:
```
➜ scala-demo scala-cli compile Hello.scala --print-class-path --dep "com.lihaoyi::pprint:0.8.1,exclude=com.lihaoyi%%%sourcecode" -v -v   
Exception in thread "main" java.lang.RuntimeException: Error parsing dependency 'com.lihaoyi::pprint:0.8.1,exclude=com.lihaoyi%%%sourcecode': Unrecognized excluded module: 'com.lihaoyi%%%sourcecode' (expected: org%name or org%%name)
	at scala.sys.package$.error(package.scala:27)
	at scala.cli.commands.shared.SharedOptions$.parseDependencies$$anonfun$3(SharedOptions.scala:651)
	at scala.collection.immutable.List.flatMap(List.scala:293)
	at scala.cli.commands.shared.SharedOptions$.parseDependencies(SharedOptions.scala:653)
	at scala.cli.commands.shared.SharedOptions.buildOptions$$anonfun$1(SharedOptions.scala:330)
	at scala.build.EitherCps$Helper.apply(EitherCps.scala:19)
	at scala.cli.commands.shared.SharedOptions.buildOptions(SharedOptions.scala:350)
	at scala.cli.commands.ScalaCommand.buildOptions$$anonfun$1(ScalaCommand.scala:246)
	at scala.Option.map(Option.scala:242)
	at scala.cli.commands.ScalaCommand.buildOptions(ScalaCommand.scala:246)
	at scala.cli.commands.ScalaCommand.run(ScalaCommand.scala:269)
	at scala.cli.commands.ScalaCommand.run(ScalaCommand.scala:266)
	at caseapp.core.app.CaseApp.main(CaseApp.scala:150)
	at caseapp.core.app.CommandsEntryPoint.main(CommandsEntryPoint.scala:120)
	at scala.cli.ScalaCliCommands.main(ScalaCliCommands.scala:125)
	at scala.cli.ScalaCli$.main0(ScalaCli.scala:198)
	at scala.cli.ScalaCli$.main(ScalaCli.scala:102)
	at scala.cli.ScalaCli.main(ScalaCli.scala)
```

Reply:
still, `exclude=org%name_2.13` would be supported for a Scala `2.13.x` dependency, right?

Reply:
weird inconsistency for triple `%` not being supported btw...